sdk/location/src/main/java/com/klaviyo/location/KlaviyoLocationManager.kt

```diff
@@ -61,6 +89,20 @@ internal object KlaviyoLocationManager : LocationManager {
         }
     }
 
+    override fun onGeofenceSync(callback: GeofenceObserver) {
```

Comment:
This observer will be the basis for
- updating monitored geofences with the system location APIs
- test app UI to display current fences